.vib/supabase/cypress/cypress/integration/supabase_spec.js

```diff
@@ -0,0 +1,143 @@
+    });
+  });
+});
+
+it('can create a schema and it appears on the list of schemas (meta)', () => {
+  // Source: https://supabase.github.io/postgres-meta/
+  cy.fixture('schemas').then((schemas) => {
+    const schemaPayload = {
+      name: `${schemas.newSchema.name}${random}`,
+      owner: schemas.newSchema.owner,
+    };
+    cy.request({
+      method: 'POST',
+      headers: { apikey: Cypress.env('serviceKey') },
+      url: '/pg/schemas',
+      body: schemaPayload,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(JSON.stringify(response.body)).to.have.string(schemaPayload.name);
+    });
+
+    cy.request({
+      method: 'GET',
+      headers: { apikey: Cypress.env('serviceKey') },
+      url: '/pg/schemas',
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(JSON.stringify(response.body)).to.have.string(schemaPayload.name);
+    });
+  });
+});
+
+it('can create a bucket and upload a file (storage)', () => {
+  // Source: https://github.com/supabase/storage-api
+  cy.fixture('buckets').then((buckets) => {
+    const bucketPayload = {
+      id: `${buckets.newBucket.name}${random}`,
+      name: `${buckets.newBucket.name}${random}`,
+      public: buckets.newBucket.public,
+    };
+    cy.request({
+      method: 'POST',
+      headers: {
+        apikey: Cypress.env('serviceKey'),
+        Authorization: `Bearer ${Cypress.env('serviceKey')}`,
+      },
+      url: '/storage/v1/bucket',
+      body: bucketPayload,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(JSON.stringify(response.body)).to.have.string(bucketPayload.id);
+    });
+
+    cy.request({
+      method: 'GET',
+      headers: {
+        apikey: Cypress.env('serviceKey'),
+        Authorization: `Bearer ${Cypress.env('serviceKey')}`,
+      },
+      url: '/storage/v1/bucket',
+      body: bucketPayload,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(JSON.stringify(response.body)).to.have.string(bucketPayload.id);
+    });
+
+    cy.fixture('files').then((files) => {
+      const filePayload = {
+        name: `${files.newFile.name}${random}`,
+        content: `${random}_${files.newFile.content}`,
+      };
+      cy.request({
+        method: 'POST',
+        headers: {
+          apikey: Cypress.env('serviceKey'),
+          Authorization: `Bearer ${Cypress.env('serviceKey')}`,
+        },
+        url: `/storage/v1/object/${bucketPayload.id}/${filePayload.name}`,
+        body: filePayload.content,
+      }).then((response) => {
+        expect(response.status).to.eq(200);
+        expect(JSON.stringify(response.body)).to.have.string(`${bucketPayload.id}/${filePayload.name}`);
+      });
+    });
+
+    cy.fixture('files').then((files) => {
+      const filePayload = {
+        name: `${files.newFile.name}${random}`,
+        content: `${random}_${files.newFile.content}`,
+      };
```

Comment:
Re-defining the `filePayload` constant here, when in reality is the same as in line 107, breaks with the pattern you have been doing in the rest of the file: a single constant (ex: `userPayload` that is then reused in upcoming `cy.request` commands). When you see it at first sight it seems that `filePayload` will hold new values, but is the same ones as before.